trade-finance-manager-ui/templates/utilisation-reports/_macros/report-reconciliation-table.njk

```diff
@@ -24,20 +25,30 @@
       </thead>
 
       <tbody class="govuk-table__body">
-        {% for summaryItem in summaryItems %}
-          {% set checkboxId = ['set-status--reportId-', summaryItem.reportId, '-currentStatus-', summaryItem.status] | join %}
-          <tr
-            class="govuk-table__row"
-            data-cy="utilisation-report-reconciliation-table-row-bank-{{ summaryItem.bank.id }}-submission-month-{{ submissionMonth
-              }}">
-            <th scope="row" class="govuk-table__header govuk-!-font-weight-regular" data-sort-value="{{ summaryItem.bank.name }}">
-            {# TODO FN-1603 - link to individual bank report page (replace `div` with `a`) #}
-            {# <a class="govuk-link govuk-link--no-visited-state" href="#">#}
+      {% for summaryItem in summaryItems %}
+        {% set checkboxId = ['set-status--reportId-', summaryItem.reportId, '-currentStatus-', summaryItem.status] | join %}
+        {% set checkboxLabel %}
```

Comment:
Possibly not directly related to your changes but I'm not sure it makes sense to call this the checkbox label given it is quite a bit more than that. I also actually can't tell but is this used a label for the checkbox but have we checked keyboard nav and screenreader usage both still make sense for following the link and also checking the check box now this can be a link?

Reply:
This was originally passed to the checkbox macro as a label for the checkbox, so it can be called something else now. I'm not fully sure what the second part of your question means, but, if I understand correctly, then that won't be a problem now that the checkbox has been moved to the end of the table row

Reply:
Sorry, I just mean have you checked the table is sensibly navigable by keyboard, and then separately if you have tried with a screenreader. I think most interesting is to check if what is announced when landing on a checkbox is meaningful and clear given the text content presumably telling you which bank it belongs to is now in a link.

Reply:
I think the checkbox doesn't have any good accessibility with it, but that probably needs to be logged as a new ticket and fixed in that. I'm going to resolve this conversation for now as I can't merge with unresolved conversations, but definitely something we should make a ticket for I think